index.html

```diff
@@ -70,6 +71,7 @@ <h3 class="mt-3 ml-3 font-bold text-indigo-100 text-xl mb-2">
 
           <div
```

Comment:
I think it's better to replace the `div` tag with `a` tag in all cards and it will fix the cursor style

Reply:
I think for accessibility using `a` tag is better too 👍